odra-core/src/call_result.rs

```diff
@@ -34,15 +34,48 @@ impl CallResult {
         }
     }
 
-    pub fn get_caller(&self) -> Address {
+    pub fn caller(&self) -> Address {
         self.caller
     }
 
-    pub fn get_gas_used(&self) -> u64 {
+    pub fn gas_used(&self) -> u64 {
         self.gas_used
     }
 
-    pub fn get_contract_address(&self) -> Address {
+    pub fn contract_address(&self) -> Address {
         self.contract_address
     }
+
+    pub fn event_names(&self) -> Vec<String> {
```

Comment:
I'd make this to `event_names(&self, contract: &Address)`
List of all event names from all contracts, without the origin is not really useful.